applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -858,6 +858,24 @@ int handle_at_agps(enum at_cmd_type cmd_type)
 					return err;
 				}
 			}
+#if defined(CONFIG_NRF_CLOUD_AGPS_FILTERED)
```

Comment:
Same thing here as with P-GPS, it would be best to always disable scheduled downloads for periodic tracking when A-GPS is used. Otherwise GNSS will do unnecessary scheduled downloads, although all needed data has already been supplied by A-GPS.

BTW, is low accuracy mode enabled intentionally for single fix? See line 854 above.

Reply:
But the code on line 870 is behind the `CONFIG_NRF_CLOUD_AGPS_FILTERED` flag. It makes sense to disable scheduled downloads always when periodic mode and A-GPS are used, not just when using filtered ephemerides.

There's no limitation for enabling low accuracy mode here, but is that something you want to do? It does not only make fixes using only three satellites possible, but it also does accept lower quality fixes. I don't think it should be enabled by default, especially when the user has no option to turn it off.

If you still want to keep low accuracy mode enabled, ideally the bits should be ORed together. As the GNSS API says, NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START should always be set. Currently that bit does not affect GNSS behavior, but if this is changed in a later MFW version, SLM would behave differently with different MFW versions.

Reply:
Sorry P-GPS disables scheduled download at line 952, unconditionally.
I don't prefer NRF_MODEM_GNSS_USE_CASE_LOW_ACCURACY. As I understand it's for faster TTFF. In good open air condition we always get fix (no A-GPS) in less than 30 seconds, then accuracy is more important than TTFF.

Reply:
Yes, P-GPS unconditionally disables scheduled downloads. But this is `handle_at_agps()`, i.e. A-GPS.

`NRF_MODEM_GNSS_USE_CASE_LOW_ACCURACY` is intended to be used in cases where it might be hard to find even four usable satellites. I think it should be used as the last resort, not by default.